test/functional/cmdLineTests/criu/src/CRIUSimpleTest.java

```diff
@@ -0,0 +1,85 @@
+/*******************************************************************************
+ * Copyright (c) 2022, 2022 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
+
+import org.eclipse.openj9.criu.CRIUSupport;
+
+public class CRIUSimpleTest {
+
+  public static void main(String args[]) {
+    /* JVM startup */
```

Comment:
you can just do

```
public static void main(String args[]) {
    System.out.println("Pre-checkpoint");

    checkPointJVM();

    System.out.println("Post-checkpoint");    
}
```

The rest is not needed

Reply:
@tajila Current grinder can print "testtest1" and "testtest2" successfully, but seems fail for `checkPointJVM();` function. Do you mean I can replace all line 32 to 84 with the code you commented? Thanks.

Reply:
no, I mean `sieveOfEratosthenes` and related things can be removed. For this test we just need to know that a checkpoint and restore can be performed